### PR TITLE
Drop dependency on gulp-util

### LINKED ISSUE #136: Replace deprecated dependency gulp-util
[`gulp-util`](https://www.npmjs.com/package/gulp-util) has been recently deprecated. Continuing to use this dependency may prevent the use of your library with the latest release of Gulp 4 so **it is important to replace `gulp-util`**.

The [README.md](https://github.com/gulpjs/gulp-util) lists alternatives for all the components so a simple replacement should be enough.

Your package is one of the most popular packages still relying on `gulp-util`, it would be good to publish a fixed version to npm as soon as possible.

See:
- https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
- https://github.com/gulpjs/gulp-util/issues/143

### PR DESCRIPTION
**Pull request by ohbarye**
Hello maintainers.

Following https://github.com/gulpjs/gulp-util/issues/143, I dropped dependency on gulp-util. 

closes #136 

Thanks!

### CODE REVIEW

**Comment by w0rm** [comment]
@ohbarye hi! Looks good to me. What should I do with the version bump? Are these changes backwards compatible?

File: package.json [hunk 1]
```diff
@@ -50,6 +51,7 @@
     "mocha": "^3.4.2",
     "nyc": "^11.0.3",
     "proxyquire": "^1.8.0",
-    "sinon": "^2.3.5"
+    "sinon": "^2.3.5",
+    "vinyl": "^2.1.0"
```
Comment (demurgos) [inline package.json]: `vinyl` should be in the `devDependencies`

**Comment by demurgos** [comment]
@w0rm 
This should be a patch update from a semver point of view.

**Comment by w0rm** [comment]
@demurgos thanks for taking a look!

File: package.json (hunk 1 above)
Comment (ohbarye) [reply package.json]: Thanks for your review, but `vinyl` is already in the `devDependencies`. 😄 

https://github.com/ohbarye/gulp-postcss/blob/6f15b5127f66ba7c360625c1b5e2fec4f51c575e/package.json#L47-L56

Comment (demurgos) [reply package.json]: You are right, I got confused.

**Comment by w0rm** [comment]
@ohbarye thanks for your contribution, I've just released v7.0.1

**Comment by ohbarye** [comment]
@w0rm Thanks for your review and merge!

**Pull request merged**